R/quosure-show.R

```diff
@@ -0,0 +1,18 @@
+#' Show the `Quosure` object
+#'
+#' Prints the `Quosure` object
+#' @param object (`Quosure`)
+#' @return nothing
+#' @importFrom methods show
+#' @examples
+#' q1 <- new_quosure(code = "print('a')", env = new.env())
+#' q1
+#' @export
+setMethod("show", "Quosure", function(object) {
+  obs <- names(as.list(object@env))
+  if (length(obs) > 0) {
+    cat(paste("A quosure object containing:", paste(obs, collapse = ", ")))
+  } else {
+    cat("A quosure object containing no objects")
+  }
+})
```

Comment:
1. for `cat()` calls we need to put `\n` at the very end
```
> 1
[1] 1
> q
A quosure object containing no objects> 2             # <- here!
[1] 2
```

2. How about utilizing `renv::print_env`? I recently discovered this and I loved that!
```
> q <- new_quosure(env(a = iris), "")
> env_print(q@env)
<environment: 0x7f8d7b0af698> [L]
Parent: <environment: package:teal.code>
Bindings:
• a: <df[,5]>
> q
A quosure object containing: a
```

Reply:
marking as resolved

PR here: #68